src/main/java/com/wanderersoftherift/wotr/WanderersOfTheRift.java

```diff
@@ -143,6 +143,18 @@ public static String translationId(String category, String item) {
         return category + "." + MODID + "." + item;
     }
 
+    /**
+     * Helper method to get a translationId string containing any mod id.
+     *
+     * @param category The category of the translationId (becomes a prefix)
+     * @param item     The ResourceLocation item
+     * @return A combination of category, the mod id and the item. e.g. if category is "item" and item is
+     *         "wotr:nosering.description" the result is "item.wotr.nosering.description"
+     */
+    public static String translationId(String category, ResourceLocation item) {
```

Comment:
Potentially this should live elsewhere (since it isn't WotR specific). Like ComponentUtil? Dunno.

Reply:
Not new, existing unchanged locations. It would greatly expand the PR again xD